rest_framework_simplejwt/authentication.py

```diff
@@ -38,7 +38,7 @@ def authenticate(self, request):
 
         validated_token = self.get_validated_token(raw_token)
 
-        return self.get_user(validated_token), None
+        return self.get_user(validated_token), raw_token
```

Comment:
Would it make more sense to return the validated token object which includes the decoded token payload and other conveniences?
```suggestion
        return self.get_user(validated_token), validated_token
```

Reply:
Sure. More detailed infos are better

Reply:
Sorry for the stupid question: you asked me to update Release Notes... it’s ok for me but i don’t know how to do It. Where should notes be addeded?

Reply:
Any update on this?